src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public abstract class Figure implements FigureArea {
+    protected String color;
+
+    public Figure() {
+        ColorSupplier color = new ColorSupplier();

```

Comment:
You're not supposed to set a random color here, but if you did, the `ColorSupplier` instance should be a constant as well.